frontend/js/src/user/charts/UserEntityChart.tsx

```diff
@@ -231,87 +283,284 @@ export default function UserEntityChart() {
             <>
               <div className="row">
                 <div className="col-xs-12">
-                  <h4 style={{ textTransform: "capitalize" }}>
-                    {terminology} count - <b>{entityCount}</b>
+                  <h4>
+                    <span style={{ textTransform: "capitalize" }}>
+                      {terminology}
+                    </span>
+                    &nbsp;count - <b>{entityCount} total</b>
+                    {entityCount > 1000 && (
+                      <span className="small">
+                        &nbsp;(showing the first 1000)
+                      </span>
+                    )}
                   </h4>
```

Comment:
We should first check if `entityCount` is defined or not, because in case of site-wide stats, it is undefined.

Reply:
That is fine, `undefined > 1000` will correctly be interpreted as `false`, perfect for our case.